modules/prebidmanagerAnalyticsAdapter.js

```diff
@@ -0,0 +1,193 @@
+import {ajaxBuilder} from '../src/ajax';
+import adapter from '../src/AnalyticsAdapter';
+import adapterManager from '../src/adapterManager';
+
+/**
+ * prebidmanagerAnalyticsAdapter.js - analytics adapter for prebidmanager
+ */
+const DEFAULT_EVENT_URL = 'https://endpoint.prebidmanager.com/endpoint'
+const analyticsType = 'endpoint';
+const analyticsName = 'Prebid Manager Analytics: ';
+
+var utils = require('src/utils');
+var CONSTANTS = require('src/constants.json');
+let ajax = ajaxBuilder(0);
+
+function generateUUID() { // Public Domain/MIT
```

Comment:
Any specific reason to not use https://github.com/prebid/Prebid.js/blob/master/src/utils.js#L96

Reply:
@jaiminpanchal27 , no any specific reason. Commited with prebid utils function.